SgmlTests/Tests.cs

```diff
@@ -16,9 +16,9 @@
 using NUnit.Framework;
 using Sgml;
 using System.Xml.Linq;
-#if WINDOWS_DESKTOP
```

Comment:
I couldn't see anywhere where `WINDOWS_DESKTOP` was defined, I assumed this wasn't intentionally disabled - and all the build-targets support `XPathDocument` so it seemed like a good idea to re-enable it for all targets.

Reply:
Yeah, thanks for removing this.  Not sure why it was there, perhaps xpath was missing from earlier .net core versions?But this adds Test_XPathDocument to SgmlTestsCore which is great.